src/transformers/commands/add_new_model_like.py

```diff
@@ -1442,7 +1442,11 @@ def get_user_input():
     # Get old model type
     valid_model_type = False
     while not valid_model_type:
-        old_model_type = input("What is the model you would like to duplicate? ")
+        old_model_type = input(
+            """What is the model you would like to duplicate? Please provide the lowercase `model_type`
+            - this can be found inside the configuration of any checkpoint of that model or in the name of the docs URL
+              (e.g. roberta):"""
```

Comment:
This will be rendered on several lines with the spaces. It's also a bit too long for the question I feel so would abbreviate in:
```suggestion
            "What is the model you would like to duplicate? Please provide the lowercase `model_type` (e.g. roberta):"
```
We can have more info in the doc page https://huggingface.co/docs/transformers/add_new_model